sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/aggregator/LongLastValueAggregator.java

```diff
@@ -26,33 +30,37 @@
  * <p>This class is internal and is hence not for public use. Its APIs are unstable and can change
  * at any time.
  */
-final class LongLastValueAggregator extends AbstractAggregator<Long> {
+final class LongLastValueAggregator extends AbstractAggregator<LongAccumulation> {
+  private final Supplier<ExemplarReservoir> reservoirBuilder;
+
   LongLastValueAggregator(
       Resource resource,
       InstrumentationLibraryInfo instrumentationLibraryInfo,
-      MetricDescriptor descriptor) {
+      MetricDescriptor descriptor,
+      Supplier<ExemplarReservoir> reservoirBuilder) {
```

Comment:
I don't think I would call these parameters "...builder". Probably just "reservoirSupplier" is a bit better.